### PR TITLE
update k8s plugin default pod templates to use new agent images

### PR DESCRIPTION
**Pull request by gabemontero**
part 2 of 2 for https://trello.com/c/YQm7osws/1493-5-jenkins-slave-scl-dep-updatesjenkinsintegration

@openshift/sig-developer-experience fyi / ptal

we'll merge after https://github.com/openshift/jenkins/pull/537 merges and new images are available on dockerhub

### CODE REVIEW

File: 2/contrib/s2i/run [hunk 1]
```diff
@@ -104,10 +104,16 @@ function migrate_slave_configuration() {
...
+  sed -i "s#<image>\(docker.io/\)\{0,1\}openshift/jenkins-agent-nodejs-8-centos7\(:.*\)\{0,1\}</image>#<image>docker.io/openshift/jenkins-agent-nodejs-8-centos7:${JENKINS_SLAVE_IMAGE_TAG}</image>#" /var/lib/jenkins/config.xml
 
   # replace registry.access.redhat.com/openshift3/jekins-slave-xxxx-rhel7[:optionaltag] with registry.access.redhat.com/openshift3/jenkins-slave-xxxx-rhel7:VersionTag
   sed -i "s#<image>registry.access.redhat.com/openshift3/jenkins-slave-maven-rhel7\(:.*\)\{0,1\}</image>#<image>registry.access.redhat.com/openshift3/jenkins-slave-maven-rhel7:${JENKINS_SLAVE_IMAGE_TAG}</image>#" /var/lib/jenkins/config.xml
-  sed -i "s#<image>registry.access.redhat.com/openshift3/jenkins-slave-nodejs-rhel7\(:.*\)\{0,1\}</image>#<image>registry.access.redhat.com/openshift3/jenkins-slave-nodejs-rhel7:${JENKINS_SLAVE_IMAGE_TAG}</image>#" /var/lib/jenkins/config.xml
+  sed -i "s#<image>registry.access.redhat.com/openshift3/jenkins-slave-nodejs-rhel7\(:.*\)\{0,1\}</image>#<image>registry.access.redhat.com/openshift3/jenkins-slave-nodejs-rhel7:${JENKINS_SLAVE_IMAGE_TAG}</image>#"
```
Comment (bparees) [inline 2/contrib/s2i/run]: ?

Comment (gabemontero) [reply 2/contrib/s2i/run]: copy / paste error 

i'll try again and push

**Comment by gabemontero** [comment]
copy/paste error fixed

**Comment by gabemontero** [comment]
gcs cred upload issue 

trying again

/retest

**Comment by gabemontero** [comment]
[(5) Jenkins slave SCL dep updates[jenkins_integration]](https://trello.com/c/YQm7osws/1493-5-jenkins-slave-scl-dep-updatesjenkinsintegration)

**Comment by bparees** [comment]
/lgtm
/hold
need to wait until https://github.com/openshift/jenkins/pull/537 has merged and pushed the new images before this can merge.

also need to make sure the brew rhel images are available, otherwise QE will be broken by this.

**Comment by gabemontero** [comment]
https://trello.com/c/3jgxMYUa/554-buildauto-add-new-rpms-and-images-to-cd-build-management-for-jenkins-maven-35-and-nodejs-8 has been opened to handle the rhel side of things

**Comment by gabemontero** [comment]
the v3.10 tags for maven-35 and nodejs-8 are available on docker hub

**Pull request merged**